packages/core/src/messagingProtocols/mls/types.ts

```diff
@@ -84,9 +83,3 @@ export interface CryptoProtocolConfig {
   /** if set to true, will use experimental proteus encryption/decryption library (core-crypto). If not set will fallback to the legacy proteus library (cryptobox) */
   proteus?: boolean;
 }
-
-export type MLSConversation = Conversation & {
```

Comment:
This one was duplicated, see https://github.com/wireapp/wire-web-packages/blob/main/packages/api-client/src/conversation/Conversation.ts#L104